src/lib/graphqlTimeoutMiddleware.ts

```diff
@@ -15,16 +15,26 @@ export function fieldFromResolveInfo(resolveInfo: GraphQLResolveInfo) {
 
 export function timeoutForField(field: GraphQLField<any, any>) {
   const fieldDirectives = field.astNode && field.astNode.directives
-  const directive = fieldDirectives && fieldDirectives.find(directive => directive.name.value === "timeout")
```

Comment:
@mzikherman - if this is a new file then something wasn't set up -- this is far wider than 80 characters.

Reply:
It's a typescript file, so maybe it wasn't getting hit by some of the pre-commit hooks?

Reply:
@orta - yup thats it: https://github.com/artsy/metaphysics/blob/master/package.json#L171